src/api/airdropApi.ts

```diff
@@ -0,0 +1,39 @@
+import { AirdropCheckWalletsSubscription } from 'app/api';
+import { WalletPayload } from 'app/consts';
+
+import api from './client';
```

Comment:
this won't be the same client as for notification, would be best to make ./client the abstraction and create an instance here client and for notification

Reply:
@marek-siemieniuk-morawski will take care of this one in different PR